src/Esri.ArcGISRuntime.Toolkit/Esri.ArcGISRuntime.Toolkit/Esri.ArcGISRuntime.Toolkit.csproj

```diff
@@ -13,7 +13,7 @@
     <AddSyntheticProjectReferencesForSolutionDependencies>false</AddSyntheticProjectReferencesForSolutionDependencies>
 	<GenerateDocumentationFile>true</GenerateDocumentationFile>
     <DocumentationFile>$(OutputPath)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
-
+    <LangVersion>7.2</LangVersion>
```

Comment:
Is this ok?  I thought it might be since we're already relying on bleeding-edge IDE capabilities.  I used it for a couple by-ref extension methods.

Reply:
Set it in Directory.Build.Props instead